### ISSUE TITLE
DX12: Hitman 2 Silent Assassin HD screen half cut

### ISSUE THREAD

**Issue by raven02** [RPCS3/rpcs3#1412]
DX12 screen right bottom half cut , possibly related to write color buffers

![hitman](https://cloud.githubusercontent.com/assets/3000282/12216726/3527ba38-b726-11e5-8e85-cb396ac676c4.jpg)

**Comment by tambry** [comment]
@raven02 Please also provide the log, the settings used and the LLE'd modules. Could you also elaborate on how it's related to write color buffers?

**Comment by B1ackDaemon** [comment]
I recently tested it too, log included in attachments
http://www.emunewz.net/forum/showthread.php?tid=162331&pid=228858#pid228858

**Comment by raven02** [comment]
This one can boot without any LLE modules and require color buffer to be enabled.

**Comment by tambry** [comment]
@raven02 Is this still an issue?

**Comment by raven02** [comment]
Yes , still happened in latest build.

**Comment by vlj** [comment]
It doesn't start at all for me

**Comment by raven02** [comment]
@vlj , let me try again on the latest build tonight .

**Comment by raven02** [comment]
Humm it looks okay .Probably @vlj you can try again by deleting the ini

**Comment by vlj** [comment]
OK have it working. 
I have several fix for the game soon.

**Comment by raven02** [comment]
Thanks @vlj

**Comment by raven02** [comment]
Checked and works property in latest build now.
